cmd/neofs-cli/modules/util/sign_session.go

```diff
@@ -71,7 +80,15 @@ func signSessionToken(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
-	err = stok.Sign(user.NewAutoIDSignerRFC6979(*pk))
+	forceIss, err := cmd.Flags().GetBool(forceIssuerFlag)
+	if err != nil {
+		return err
+	}
+	if forceIss {
+		err = stok.Sign(user.NewAutoIDSignerRFC6979(*pk))
+	} else {
+		err = stok.SetSignature(neofsecdsa.SignerRFC6979(*pk))
```

Comment:
check already attached issuer? warn/err if does not match/missing?

Reply:
i wouldnt add one more key<->user relation. And error is bad here to me cuz this utility should help, not limit. Dont think warning will be useful too: if other issuer is a problem - user will get his error later anyway, if not - there is nothing to worry bout